### PR TITLE
Update readme for watch mode

### PR DESCRIPTION
**Pull request by mlucool**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,6 +120,11 @@ jlpm run build
 jupyter lab build
 ```
 
+Or start in watch mode. Then after ever change, refresh the page and click "build".
+```bash
+jlpm run watch
+```
```
Comment (jaipreet-s) [inline README.md]: How about also documenting the steps to launch jupyterlab in watch mode?

```
jupyter-lab --watch
```

Comment (mlucool) [reply README.md]: Done. Watch is better but somewhat frustrating due to the time it takes so I don't always know if my change is really there.

File: README.md [hunk 2]
```diff
@@ -120,6 +120,16 @@ jlpm run build
 jupyter lab build
 ```
 
+Or start jupyter in watch mode:
+```bash
+jupyter-lab --watch
```
Comment (fcollonval) [inline README.md]:
```suggestion
jupyter lab --watch
```

This is the recommended syntax.

Comment (mlucool) [reply README.md]: My bad, old habits. It was needed before this:
https://github.com/jupyter/jupyter_core/pull/148/files#diff-043250408cc76a63bbd3f7cd15e50cdbR122

Comment (fcollonval) [reply README.md]: No problem thanks for contributing 😉

**Pull request merged**